File: scripts/temp_fix.py
import re

with open(r'c:\Users\monst\.gemini\antigravity\scratch\Sales app\scripts\create_rls_policies.sql', 'r', encoding='utf-8') as f:
    content = f.read()

# Find all CREATE POLICY lines and insert DROP POLICY IF EXISTS before them
def replace_func(match):
    policy_name = match.group(1)
    table_name = match.group(2)
    return f'DROP POLICY IF EXISTS {policy_name} ON {table_name};\n{match.group(0)}'

new_content = re.sub(r'CREATE POLICY (\"\w[^\"]+\") ON (\w+) FOR', replace_func, content)

with open(r'c:\Users\monst\.gemini\antigravity\scratch\Sales app\scripts\create_rls_policies.sql', 'w', encoding='utf-8') as f:
    f.write(new_content)

print("Success")

File: scripts/temp_fix.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'create_rls_policies.sql');
let content = fs.readFileSync(filePath, 'utf-8');

content = content.replace(/CREATE POLICY ("[^"]+") ON (\w+) FOR/g, (match, policyName, tableName) => {
    return `DROP POLICY IF EXISTS ${policyName} ON ${tableName};\n${match}`;
});

fs.writeFileSync(filePath, content, 'utf-8');
console.log('Success');
